### ISSUE TITLE
Downloading the image does not working

### ISSUE THREAD

**Issue by Hwan-seok** [GetStream/stream-chat-flutter#1228]
### Which packages are you using?

stream_chat_flutter

### On what platforms did you experience the issue?

iOS

### What version are you using?

4.3.0

### What happened?

When trying to download the image as follows, the error came out and indefinitely hangs out
> `StreamMessageListView > image tile > action button > download`

### Steps to reproduce

```bash
1. Create channel with two members (1:1)
2. Send image from User A to B
3. B opens the image page by tapping the image-containing message.
4. Tap action button(more_dots). It will came out with dropdown overlay.
5. Tap download.
6. The loading modal came out indefinitely with error and there is no way to close this.
```

### Relevant log output

```shell
[+4265 ms] [Generic] Video /var/mobile/Containers/Data/Application/3C9A6520-5E84-482A-AE20-C3BB6184C7A0/Library/Caches/28f9f64d-3cba-4f57-b7c0-090162724321.e30787ca-aa8a-44dc-a1cf-138fac53d9e7.1656305615.808928_o_IMG_0326.HEIC cannot be saved to the photo library: Error Domain=AVFoundationErrorDomain Code=-11828 "열 수 없음" UserInfo={NSUnderlyingError=0x2812d3ea0 {Error Domain=NSOSStatusErrorDomain Code=-12847 "(null)"}, NSLocalizedFailureReason=이 미디어 포맷은 지원되지 않습니다., NSURL=file:///var/mobile/Containers/Data/Application/3C9A6520-5E84-482A-AE20-C3BB6184C7A0/Library/Caches/28f9f64d-3cba-4f57-b7c0-090162724321.e30787ca-aa8a-44dc-a1cf-138fac53d9e7.1656305615.808928_o_IMG_0326.HEIC, NSLocalizedDescription=열 수 없음}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by imtoori** [comment]
Hi 
I'm trying to reproduce this, but it's working fine for me
Does it happen with every image? Can you give me more info on this?

**Comment by BrunoJurkovic** [comment]
> Hi I'm trying to reproduce this, but it's working fine for me Does it happen with every image? Can you give me more info on this?

Hey, thanks for the quick reply!

It seems to happen on images both sent from the current user & images sent by other users.

It basically hangs at the exact moment that the download is finished. It's also an issue because there is no way to exit this screen, no timeout, no anything except to close the app & reopen it.

<img width="431" alt="image" src="https://user-images.githubusercontent.com/28235257/180423819-50f93c74-16d6-4559-8d86-755b31d8abe3.png">

Is there a way to maybe temporarily disable this? If only until it is fixed?

**Comment by imtoori** [comment]
ok I managed to reproduce it and I found out it's related to this issue on the package we're using https://github.com/hui-z/image_gallery_saver/issues/204
I'm trying to find a solution, I'll keep you posted

**Comment by imtoori** [comment]
ok I found a way of fixing it, I'll push a pr asap

**Comment by pestizd** [comment]
Great tnx a lot for the quick response!

**Comment by imtoori** [comment]
closed by #1280
